index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+<section class="toolbar">
+  <div class="toolbar logo"><img src="images/logo.png" alt=""></div>
+  <div class="toolbar menu"><img src="images/menu.png" alt=""></div>
+</section>
+<section class ="container">
+     <div class="container intro">I'm Victor,
+       Glad to see you!</div>
+          <div class="container content">
+           Hello! I am a software developer! I can help you build a product, feature or website.
+           Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me
+          </div>
+     <div class="container social">LET'S CONNECT</div>
+     <div class="container twit"><img src="images/twitter.png" alt=""></div>
+     <div class="container linked"><img src="images/linkedin.png" alt=""></div>  
```

Comment:
- [ ] Great job adding the social icons! Let's make it even better by using `ul` , `li` and `a` tags , as this is a better way to create lists in HTML. Don't forget to add the missing icons.